inception-project/inception
# #1050 Populate Evaluation Result Object

**What's in the PR**
The evaluation object returned by the evaluate method of the recommenders created in #653 should be populated with statistics of the evaluation. We decided on macro-averaged precision, recall, f1 and accuracy as scores.

* closes https://github.com/inception-project/inception/issues/1050, https://github.com/inception-project/inception/issues/751

**How to test manually**
* Enable recommender learning curve visualization in srv/inception/settings.properties with show.learning.curve.diagram=true
* Enable one or more recommenders for the project
* Observe F1-score curves during annotation

**Unrelated**
* Updates the release documentation with note on updating milestones after release.

**Automatic testing**

**Documentation**
